typings/index.d.ts

```diff
@@ -0,0 +1,133 @@
+import {
+    CSSProperties,
+    CSSPropertiesArray,
+    CSSPropertiesComplete,
+    CSSPropertiesLossy,
+    CSSPropertiesPseudo,
+    CSSPropertiesRecursive,
+    CSSWideKeyword,
+} from './css-properties';
+
+export {
+    CSSProperties,
+    CSSPropertiesArray,
+    CSSPropertiesComplete,
+    CSSPropertiesLossy,
+    CSSPropertiesPseudo,
+    CSSPropertiesRecursive,
+    CSSWideKeyword,
+};
+
+/**
+ * Aphrodite style declaration
+ */
+export type StyleDeclarationMap = Map<keyof CSSProperties, string | number>;
+export type StyleDeclaration<T = {}> = {
+    [P in keyof T]: CSSProperties | StyleDeclarationMap

```

Comment:
I didn't realize that aphrodite now explicitly supports calls like:

```
const declMap = new Map()
declMap.set("background", "red");
StyleSheet.create({
  foo: declMap
})
```

Does that seem right to you, @lencioni?

Reply:
Yep, this was added mostly to allow developers to ensure the order of styles in older versions of node. We needed it at Airbnb when it was introduced, but since then we have been able to update our node version to a new enough version that preserves key order in objects so we no longer need this feature. In practice, I'm not sure how many people are using it anymore.